### ISSUE TITLE
RSS feed for cheching new releases

### ISSUE THREAD

**Issue by loishii** [erengy/taiga#85]
I believe I may have found a problem with the torrent checker, I'm not entirely sure.
I recently installed Taiga and was experimenting with the torrent rss, trying to put filters to find and download the animes I'm watching, but looks like some episodes are not showing up on Taiga.

For exemple: I was looking for Saenai Heroine no Sodatekata and I found it on Tokyotosho, but even if it shows up on tokyotosho rss, it doesn't shows up on Taiga.

I have disabled all filters on Taiga and tested with the same tokyotosho link:

1) tokyotosho rss:
https://i.imgur.com/lU4sqfY.jpg

2) tokyotosho rss loaded on utorrent:
https://i.imgur.com/B9hB4vm.jpg

3) tokyotosho rss on Taiga:
https://i.imgur.com/A3LBzi0.jpg
I tried sorting it by name and even by fansub group and I couldn't find the Doki release of Saenai Heroine no Sodatekata.

Sorry if this isn't actually a problem and I'm just dumb, but my conclusion was that if it shows up on the browser and even shows up on utorrent when loaded, the problem would possibly be on Taiga.

**Comment by erengy** [comment]
I can see those releases with the same URL, so I'm not sure why they don't show up for you:
https://i.imgur.com/IkJKGI5.png

Do you still have the same issue? Are you using an older or modified version of Taiga? Are you sure you've disabled all filters (i.e. *Settings → Torrents → Filters → Enable torrent filters*)? To make sure Taiga's getting the same feed, you can check `Taiga\data\feed\dG9reW90b3Noby5pbmZv\feed.xml` manually.

**Comment by loishii** [comment]
Thanks for the reply.
I'm using Taiga version 1.1.11, downloaded directly from http://taiga.erengy.com/
I tried again making sure the filters were off but still got the same results.

However, when I checked the .xml file, I couldn't find the episode on it.
https://i.imgur.com/bjyCfNf.jpg
I tried searching for others terms, but for some weird reason that is the only episode I couldn't find.

Would that be the problem? If so, how can I fix it?

**Comment by erengy** [comment]
That `feed.xml` seems a bit outdated. Its `pubDate` is `Mon, 02 Mar 2015 05:18:08 GMT`, while the Doki release is published at `Mon, 02 Mar 2015 09:13:34 GMT`, about 4 hours later.

Does the `feed.xml` get updated when you hit the *Check new torrents* button? You can also try running Taiga in [debug mode](https://github.com/erengy/taiga/wiki/How-to-run-Taiga-in-debug-mode) and examine the output to see if the latest RSS feed got transferred or not.

Oh. It may be related to the fact that you've installed the application under Program Files directory. Since Taiga doesn't require administrator rights to run, User Account Control may prevent Taiga from overwriting `feed.xml`, and any other file under the `data` directory, for that matter. If that's the case, you should either move Taiga to the default installation directory (which is `%AppData%\Taiga`), or run it as administrator.

**Comment by loishii** [comment]
That was it, administrator mode.
I did install Taiga running the installation with administrator mode, but after that I didn't open Taiga with administrator mode.
Now I opened Taiga with administrator mode, clicked on "Check new torrents" button and done, the list was updated. Now I can see the episode on the list.

I installed on Program Files directory just because I like having all my installed programs on the same place, never thought that it would have a problem with the program permissions.
I will move it to the default installation as suggested.

Thank you very much :)
